### ISSUE TITLE
How to process with base64 encoded JSON data?

### ISSUE THREAD

**Issue by cxplay** [AdguardTeam/CoreLibs#1814]
I'm analyzing an Android app and noticed that its server returns a base64-encoded JSON response to it. Is there a way to handle this encoded JSON data via AdGuard? I was using jsonprune to process regular JSON, and now the situation seems to be getting more complicated.

**Comment by ameshkov** [comment]
I actually don't think there's any option to do that at the moment.

**Comment by cxplay** [comment]
> I actually don't think there's any option to do that at the moment.

Indeed. I think if AdGuard had modifier rules that could pre-decode the response body in some kind of encoding it might be possible to solve this problem.
